### ISSUE TITLE
Create an IIIF manifest

### ISSUE THREAD

**Issue by jamiefolsom** [performant-software/digital-fauvel#3]
### How to test

1. Go to: http://projectmirador.org/demo/
2. Click the button at top left of the left hand image which appears like a 2x2 grid, as pictured here

<img width="461" alt="screenshot 2017-09-01 11 41 47" src="https://user-images.githubusercontent.com/882377/29977991-61075580-8f0d-11e7-8193-f82717b3f24f.png">

3. Click "Replace Object"
4. Into the "Add new object from URL" field, at top right, paste: http://fauvel.archivengine.com/manifest.json
5. You should see: "Roman de Fauvel" appear at top of the list of manifests. Click any image to view it.
6. If that works, and you see 90-some odd images at bottom as thumbnails, all of which work too, this should be accepted.

### Questions for Anna and/or Rebecca

Regarding manifest metadata:

- [ ] What should the "label" field contain?
- [ ] What should the "description" field contain be?
- [ ] What should the "attribution" field contain? Anna mentioned Gallica.
- [ ] What license should we use? This is the one the Bodleian Manifest Editor slots in by default: [cc by](https://creativecommons.org/licenses/by/3.0/). There are others, like [cc by-nc-sa](https://creativecommons.org/licenses/by-nc-sa/4.0/)

Regarding per-image metadata:

- [ ] What should the "label" field contain? "Page 50" for 50.tiff, for example?
- [ ] Do we know of any other metadata fields we're going to want for sure on every image?
- [ ] Do we have that data in hand?

NB: These values can be left blank for now, and/or changed easily at a later date.

**Comment by jamiefolsom** [comment]
Hi @fiebrink1! This is the issue I thought you and Anna might like to chime in on.

**Comment by annazaya** [comment]
Hi @jamiefolsom abd @fiebrink1 ! I'm here. Couldn't get the "Add a new object from URL" to work, though.

**Comment by fiebrink1** [comment]
Same here - got error in my Chrome console:

XMLHttpRequest cannot load https://performant-software.github.io/digital-fauvel/manifest.json. Redirect from 'https://performant-software.github.io/digital-fauvel/manifest.json' to 'http://fauvel.archivengine.com/manifest.json' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://projectmirador.org' is therefore not allowed access.

**Comment by SteveMarvin** [comment]
Yeah, I'm getting those, too. Interestingly, the **first** time I did it (just before Anna posted above), I was able to log into the page, add the json URL, and see the manifest, but on reload that manifest is gone, and I have the same behavior that they are seeing above. I can load the page, but it generates errors on load, and the "Add a new object from URL" control is unresponsive. 

So it worked great...once!

Errors pasted below:
![screen shot 2017-09-05 at 10 38 36 am](https://user-images.githubusercontent.com/18423533/30066690-a07fe1d4-9226-11e7-882d-acb8966175a5.png)
![screen shot 2017-09-05 at 10 38 43 am](https://user-images.githubusercontent.com/18423533/30066704-a5dcef8c-9226-11e7-8ae7-38de16b8d3eb.png)
![screen shot 2017-09-05 at 10 38 52 am](https://user-images.githubusercontent.com/18423533/30066703-a5cf19b6-9226-11e7-848e-4dd1195a3091.png)
![screen shot 2017-09-05 at 10 39 00 am](https://user-images.githubusercontent.com/18423533/30066707-a8674d7e-9226-11e7-974a-bdddbca693eb.png)

**Comment by jamiefolsom** [comment]
Hi @SteveMarvin -- my mistake -- I moved the manifest. It's now at: http://fauvel.archivengine.com/manifest.json Sorry about that!

**Comment by SteveMarvin** [comment]
OK, can now add at will. I note that it is not retained, and is removed from the list on refresh, but can then be added again via URL. 👍

**Comment by jamiefolsom** [comment]
Great, that's as expected. Closing, thanks!

**Comment by jamiefolsom** [comment]
As mentioned in #1: that missing locale file is a good catch -- will be addressed as a separate issue. Thanks, @SteveMarvin!
